index.js

```diff
@@ -4,9 +4,46 @@ const closeNavButton = document.getElementById("close-nav-button");
 const addTaskButton = document.getElementById("add-task-button");
 const taskTitleInput = document.getElementById("task-title-input");
 const taskList = document.getElementById("task-list");
+const taskSortSection = document.getElementById("sort-tasks");
+const sreachInput = document.getElementById("search-input");
+const searchButton = document.getElementById("search-button");
+const timeFilterSection = document.getElementById("filter-by-time-tasks");
 
 const sideMenuContainerWidth = "200px";
-const tasks = [];
+
+let tasks = [
+  {
+    id: 22333,
+    title: "lastDaaaay",
+    isDone: false,
+    createdAt: "2022-09-06T11:45:05.267Z",
+  },
+  {
+    id: 225633,
+    title: "totooooday",
+    isDone: false,
+    createdAt: "2022-09-07T11:45:05.267Z",
+  },
+  {
+    id: 2256733,
+    title: "lastWeeeeekkk",
+    isDone: false,
+    createdAt: "2022-09-02T11:45:05.267Z",
+  },
+  {
+    id: 57533,
+    title: "lastMonnni",
+    isDone: false,
+    createdAt: "2022-08-16T11:45:05.267Z",
+  },
+  {
+    id: 22433,
+    title: "day day past",
+    isDone: false,
+    createdAt: "2022-09-06T11:45:05.267Z",
+  },
+];
+let allTasks = [];
```

Comment:
`tasks` should contain of all tasks. This leads to confusion.

Reply:
OK